# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsGlobalWindowInner.cpp
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 dom/base/nsGlobalWindowInner.cpp
--- a/dom/base/nsGlobalWindowInner.cpp	Wed Jun 11 13:42:13 2025 +0000
+++ b/dom/base/nsGlobalWindowInner.cpp	Wed Jun 11 18:04:27 2025 +0300
@@ -183,7 +183,6 @@
 #include "mozilla/dom/VRDisplayEventBinding.h"
 #include "mozilla/dom/VREventObserver.h"
 #include "mozilla/dom/VisualViewport.h"
-#include "mozilla/dom/WebIdentityHandler.h"
 #include "mozilla/dom/WebIDLGlobalNameHash.h"
 #include "mozilla/dom/WindowBinding.h"
 #include "mozilla/dom/WindowContext.h"
@@ -1444,7 +1443,6 @@
   NS_IMPL_CYCLE_COLLECTION_TRAVERSE(mDocumentCsp)
   NS_IMPL_CYCLE_COLLECTION_TRAVERSE(mBrowserChild)
   NS_IMPL_CYCLE_COLLECTION_TRAVERSE(mDoc)
-  NS_IMPL_CYCLE_COLLECTION_TRAVERSE(mWebIdentityHandler)
 
   NS_IMPL_CYCLE_COLLECTION_TRAVERSE(mIdleRequestExecutor)
   for (IdleRequest* request : tmp->mIdleRequestCallbacks) {
@@ -7791,18 +7789,6 @@
   return wc->GetUsingStorageAccess();
 }
 
-WebIdentityHandler* nsPIDOMWindowInner::GetOrCreateWebIdentityHandler() {
-  if (mWebIdentityHandler) {
-    return mWebIdentityHandler;
-  }
-  mWebIdentityHandler = new WebIdentityHandler(this);
-  bool success = mWebIdentityHandler->MaybeCreateActor();
-  if (!success) {
-    mWebIdentityHandler = nullptr;
-  }
-  return mWebIdentityHandler;
-}
-
 CloseWatcherManager* nsPIDOMWindowInner::EnsureCloseWatcherManager() {
   if (!mCloseWatcherManager) {
     mCloseWatcherManager = new CloseWatcherManager();